plugin/src/Reconciler/reify.lua

```diff
@@ -40,6 +42,20 @@ local function addAllToPatch(patchSet, virtualInstances, id)
 	end
 end
 
+local function createInstance(instanceMap, className, properties)
+	-- Custom logic for MeshParts
+	-- see: https://github.com/rojo-rbx/rojo/issues/402
+	if className == "MeshPart" then
+		local okMeshId, meshId = decodeValue(properties.MeshId, instanceMap)
+
+		-- It is okay to provide default properties here, because 
+		-- they will be overriden on the update cycle
+		return InsertService:CreateMeshPartAsync(meshId, Enum.CollisionFidelity.Default, Enum.RenderFidelity.Automatic)
```

Comment:
This is the first time we'll be introducing an async function into Rojo's reconciler bits. This is a possible source of bugs that we should watch out for in the future.